src/tasks.md

```diff
@@ -0,0 +1,255 @@
+# Common Tasks
+
+This page lists a set of tasks that can be performed after you [install Kubewarden](/quick-start.html#install) in your Kubernetes cluster.
+
+Each task can be done separately; however, if you're not familiar with Kubewarden, or Kubernetes policies in general, we recommend that you follow the tasks below in sequential order.
+
+## Test Policies
+
+Kubewarden has two main tools to help you find policies and test them locally:
+
+- [Kubewarden Policy Hub](https://hub.kubewarden.io/)
+- [`kwctl`](https://github.com/kubewarden/kwctl) CLI tool
+
+### Kubewarden Policy Hub
+
+The Kubewarden Policy Hub hosts policies contributed by the community. For example, you can find substitutes to the [deprecated Kubernetes Pod Security Policies](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/), created by the Kubewarden developers. 
+
+As shown in the picture below, once you find the policy to be tested, you can copy the registry path<sup>1</sup> or download<sup>2</sup> the `WASM` binary containing the policy and additional metadata:
```

Comment:
Nitpicking, `Wasm` is written with only the first letter uppercase (see https://webassembly.org/)

```suggestion
As shown in the picture below, once you find the policy to be tested, you can copy the registry path<sup>1</sup> or download<sup>2</sup> the `Wasm` binary containing the policy and additional metadata:
```